integration/common_test.go

```diff
@@ -36,16 +36,21 @@ func newKubeTest() (*harness.Test, error) {
 	return test, nil
 }
 
-func eksctl(args ...string) *gexec.Session {
-	command := exec.Command(eksctlPath, args...)
-	command.Env = os.Environ()
-	command.Env = append(command.Env, "EKSCTL_EXPERIMENTAL=true")
-	fmt.Fprintf(GinkgoWriter, "calling %q with %s and %v\n", eksctlPath, "EKSCTL_EXPERIMENTAL=true", args)
+type params struct {
```

Comment:
Why is this an unexported struct with exported fields?

Reply:
No specific reason, what would you suggest instead?